src/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Gregorio configuration headers. Copyright (C) 2015 Elie Roux
```

Comment:
How about `Élie Roux and the Gregorio Project authors`.  Then at least your name is on it to lend legal legitimacy?